llvm/utils/git/requirements.txt.in

```diff
@@ -3,11 +3,5 @@
 # pip install pip-tools
 # pip-compile -o requirements.txt  requirements.txt.in
 
-<<<<<<< HEAD
 PyGithub
 #GitPython==3.1.29
-=======
```

Comment:
please use the set from Pulldown which has latest setting.

Reply:
After a further look, line8 has been commented in SYCLomatic, we only keep line7.

Reply:
LGTM